### ISSUE TITLE
failed: [vpm135.front.sepia.ceph.com] (item=ppa:gluster/libntirpc) 

### ISSUE THREAD

**Issue by vasukulkarni** [ceph/ceph-ansible#2286]
using stable-2.1 branch to install jewel

http://qa-proxy.ceph.com/teuthology/vasu-2017-12-19_21:10:25-ceph-ansible-jewel-distro-basic-vps/1982447/teuthology.log

```
2017-12-19T21:28:31.658 INFO:teuthology.orchestra.run.vpm021.stdout:TASK [ceph.ceph-common : add gluster nfs ganesha repo] *************************
2017-12-19T21:28:31.658 INFO:teuthology.orchestra.run.vpm021.stdout:task path: /home/ubuntu/ceph-ansible/roles/ceph-common/tasks/installs/debian_ceph_repository.yml:55
2017-12-19T21:28:33.315 INFO:teuthology.orchestra.run.vpm021.stdout:failed: [vpm135.front.sepia.ceph.com] (item=ppa:gluster/libntirpc) => {
2017-12-19T21:28:33.316 INFO:teuthology.orchestra.run.vpm021.stdout:    "changed": false,
2017-12-19T21:28:33.316 INFO:teuthology.orchestra.run.vpm021.stdout:    "failed": true,
2017-12-19T21:28:33.316 INFO:teuthology.orchestra.run.vpm021.stdout:    "item": "ppa:gluster/libntirpc"
2017-12-19T21:28:33.316 INFO:teuthology.orchestra.run.vpm021.stdout:}
2017-12-19T21:28:33.317 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:33.317 INFO:teuthology.orchestra.run.vpm021.stdout:MSG:
2017-12-19T21:28:33.317 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:33.317 INFO:teuthology.orchestra.run.vpm021.stdout:failed to fetch PPA information, error was: HTTP Error 404: Not Found
2017-12-19T21:28:33.318 INFO:teuthology.orchestra.run.vpm021.stdout:failed: [vpm195.front.sepia.ceph.com] (item=ppa:gluster/libntirpc) => {
2017-12-19T21:28:33.318 INFO:teuthology.orchestra.run.vpm021.stdout:    "changed": false,
2017-12-19T21:28:33.318 INFO:teuthology.orchestra.run.vpm021.stdout:    "failed": true,
2017-12-19T21:28:33.318 INFO:teuthology.orchestra.run.vpm021.stdout:    "item": "ppa:gluster/libntirpc"
2017-12-19T21:28:33.319 INFO:teuthology.orchestra.run.vpm021.stdout:}
2017-12-19T21:28:33.319 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:33.319 INFO:teuthology.orchestra.run.vpm021.stdout:MSG:
2017-12-19T21:28:33.319 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:33.320 INFO:teuthology.orchestra.run.vpm021.stdout:failed to fetch PPA information, error was: HTTP Error 404: Not Found
2017-12-19T21:28:33.368 INFO:teuthology.orchestra.run.vpm021.stdout:failed: [vpm021.front.sepia.ceph.com] (item=ppa:gluster/libntirpc) => {
2017-12-19T21:28:33.368 INFO:teuthology.orchestra.run.vpm021.stdout:    "changed": false,
2017-12-19T21:28:33.369 INFO:teuthology.orchestra.run.vpm021.stdout:    "failed": true,
2017-12-19T21:28:33.369 INFO:teuthology.orchestra.run.vpm021.stdout:    "item": "ppa:gluster/libntirpc"
2017-12-19T21:28:33.369 INFO:teuthology.orchestra.run.vpm021.stdout:}
2017-12-19T21:28:33.369 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:33.369 INFO:teuthology.orchestra.run.vpm021.stdout:MSG:
2017-12-19T21:28:33.369 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:33.370 INFO:teuthology.orchestra.run.vpm021.stdout:failed to fetch PPA information, error was: HTTP Error 404: Not Found
2017-12-19T21:28:34.842 INFO:teuthology.orchestra.run.vpm021.stdout:failed: [vpm021.front.sepia.ceph.com] (item=ppa:gluster/nfs-ganesha) => {
2017-12-19T21:28:34.842 INFO:teuthology.orchestra.run.vpm021.stdout:    "changed": false,
2017-12-19T21:28:34.843 INFO:teuthology.orchestra.run.vpm021.stdout:    "failed": true,
2017-12-19T21:28:34.843 INFO:teuthology.orchestra.run.vpm021.stdout:    "item": "ppa:gluster/nfs-ganesha"
2017-12-19T21:28:34.843 INFO:teuthology.orchestra.run.vpm021.stdout:}
2017-12-19T21:28:34.843 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:34.843 INFO:teuthology.orchestra.run.vpm021.stdout:MSG:
2017-12-19T21:28:34.843 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:34.844 INFO:teuthology.orchestra.run.vpm021.stdout:failed to fetch PPA information, error was: HTTP Error 404: Not Found
2017-12-19T21:28:34.861 INFO:teuthology.orchestra.run.vpm021.stdout:failed: [vpm135.front.sepia.ceph.com] (item=ppa:gluster/nfs-ganesha) => {
2017-12-19T21:28:34.861 INFO:teuthology.orchestra.run.vpm021.stdout:    "changed": false,
2017-12-19T21:28:34.861 INFO:teuthology.orchestra.run.vpm021.stdout:    "failed": true,
2017-12-19T21:28:34.861 INFO:teuthology.orchestra.run.vpm021.stdout:    "item": "ppa:gluster/nfs-ganesha"
2017-12-19T21:28:34.861 INFO:teuthology.orchestra.run.vpm021.stdout:}
2017-12-19T21:28:34.862 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:34.862 INFO:teuthology.orchestra.run.vpm021.stdout:MSG:
2017-12-19T21:28:34.862 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:34.862 INFO:teuthology.orchestra.run.vpm021.stdout:failed to fetch PPA information, error was: HTTP Error 404: Not Found
2017-12-19T21:28:34.862 INFO:teuthology.orchestra.run.vpm021.stdout:failed: [vpm195.front.sepia.ceph.com] (item=ppa:gluster/nfs-ganesha) => {
2017-12-19T21:28:34.862 INFO:teuthology.orchestra.run.vpm021.stdout:    "changed": false,
2017-12-19T21:28:34.863 INFO:teuthology.orchestra.run.vpm021.stdout:    "failed": true,
2017-12-19T21:28:34.863 INFO:teuthology.orchestra.run.vpm021.stdout:    "item": "ppa:gluster/nfs-ganesha"
2017-12-19T21:28:34.863 INFO:teuthology.orchestra.run.vpm021.stdout:}
2017-12-19T21:28:34.863 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:34.863 INFO:teuthology.orchestra.run.vpm021.stdout:MSG:
2017-12-19T21:28:34.863 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:34.864 INFO:teuthology.orchestra.run.vpm021.stdout:failed to fetch PPA information, error was: HTTP Error 404: Not Found
2017-12-19T21:28:34.864 INFO:teuthology.orchestra.run.vpm021.stdout:	to retry, use: --limit @/home/ubuntu/ceph-ansible/site.retry
2017-12-19T21:28:34.864 INFO:teuthology.orchestra.run.vpm021.stdout:
2017-12-19T21:28:34.864 INFO:teuthology.orchestra.run.vpm021.stdout:PLAY RECAP *********************************************************************
2017-12-19T21:28:34.864 INFO:teuthology.orchestra.run.vpm021.stdout:vpm021.front.sepia.ceph.com : ok=16   changed=3    unreachable=0    failed=1
2017-12-19T21:28:34.865 INFO:teuthology.orchestra.run.vpm021.stdout:vpm135.front.sepia.ceph.com : ok=16   changed=3    unreachable=0    failed=1
2017-12-19T21:28:34.865 INFO:teuthology.orchestra.run.vpm021.stdout:vpm195.front.sepia.ceph.com : ok=16   changed=3    unreachable=0    failed=1
```

**Comment by leseb** [comment]
Looks like a ppa error, nothing related to ceph-ansible...

**Comment by leseb** [comment]
@vasukulkarni can we close this? or should we change the ppa config?

**Comment by vasukulkarni** [comment]
@leseb I think you should remove this item from the config 'ppa:gluster/libntirpc' , I tried again to see if it was intermittent issue, but seems like it doesn't exist anymore. so anyone using stable-2.1 on xenial would be unable to configure cluster.

@djgalloway  FYI

**Comment by leseb** [comment]
@vasukulkarni fixed in stable-2.1
